project/plugins.sbt

```diff
@@ -6,14 +6,17 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
 
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.15.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.20.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.2.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.5.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.18")
+addSbtPlugin("org.playframework"  % "sbt-plugin"  % "3.0.1")
 
 addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.5.1")
```

Comment:
This plugin needs to be updated to addSbtPlugin("io.github.irundaia" % "sbt-sassify" % "1.5.2")